src/_guides/libraries/c-interop.md

```diff
@@ -12,16 +12,11 @@ _FFI_ stands for [_foreign function interface._][FFI]
 Other terms for similar functionality include _native interface_
 and _language bindings._
 
-{{site.alert.warning}}
-The dart:ffi library is [in active development][ffi issue]
-and isn't complete yet.
-The API is likely to have breaking changes
-between now and its completion.
-To use dart:fii, you need a 2.6 prerelease of the Dart SDK.
-If you're developing a Flutter app, use the Flutter dev channel,
-as described in the [Flutter dart:ffi page.][binding]
-Otherwise, use the
-[Dart dev channel](/get-dart#about-release-channels-and-version-strings).
+{{site.alert.info}}
+  As of Dart 2.6, [dart:ffi is in beta.][ffi issue]
```

Comment:
We should make the wording consistent with the library info maybe, this is the wording shipped with D26.

```dart
/**
 * Foreign Function Interface for interoperability with the C programming language.
 *
 * **NOTE**: Dart:FFI is in technical preview. The overall feature is incomplete,
 * may contain issues, and breaking API changes are still expected.
 *
 * For further details, please see: https://dart.dev/server/c-interop
 *
 * {@category VM}
 */
```

Reply:
In `sdk/lib/ffi/ffi.dart`.

Reply:
Fixing in #2067

Reply:
I've also updated https://github.com/flutter/website/pull/3192 to match. But we won't merge that PR until we confirm that flutter dev channel is using 2.6.